style.css

```diff
@@ -274,3 +284,142 @@ textarea {
   text-decoration: none;
   color: var(--dark);
 }
+
+/* Desktop Styles min width 720px */
+@media screen and (min-width: 768px) {
+  .nav__logo,
+  .nav__hamburger {
+    display: none;
+  }
+
+  .nav__links {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+  }
+
+  .nav__links-items {
+    list-style: none;
+    display: flex;
+  }
+
+  .nav__links-items a {
+    color: var(--dark);
+    font-weight: bold;
+  }
+
+  header {
+    background-image: url(./images/header-illustration-desktop.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: top;
+  }
+
+  .hero {
+    background-image: none;
+  }
+
+  .hero .container {
+    display: flex;
+    padding: 8rem 0 12rem 0;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .hero__title,
+  .hero__content {
+    width: 60%;
+  }
+
+  .work__heading {
+    padding-top: 4rem;
+  }
+
+  .work__heading-recent,
+  .work__heading-work {
+    display: inline-block;
+  }
+
+  .work__heading-work::before {
+    display: none;
+  }
+}
+
+/* Desktop styles */
+@media screen and (min-width: 840px) {
+  .projects {
+    grid-template-columns: repeat(2, 1fr);
+    gap: 2rem;
+  }
+
+  .about {
+    background: url(./images/illustration-about-me-2.svg) 0 50% no-repeat, url(./images/illustration-about-me.svg) 100% 50% no-repeat;
```

Comment:
- [ ] The same thing here. 🧐️

**Your Design**

![Screenshot from 2023-01-20 09-03-27](https://user-images.githubusercontent.com/17243325/213630462-41ec6449-c5a4-4e97-8bce-8bcf5eb07dc3.png)